### PR TITLE
Add azure archive location type to the config

### PR DESCRIPTION
**Pull request by KillianG**
The purpose of this PR is to add azure archive location type to the config
Issue: BB-359

### CODE REVIEW

**Review by welansari — APPROVED** [review]

File: lib/Config.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ const locationTypeMatch = {
     'location-scality-artesca-s3-v1': 'aws_s3',
     'location-ceph-radosgw-s3-v1': 'aws_s3',
     'location-dmf-v1': 'dmf',
+    'location-azure-archive-v1': 'azure_archive'
```
Comment (francoisferrand) [inline lib/Config.js:24]: seeing the extent of this change, i wonder what/how this value is used...

Comment (KillianG) [reply lib/Config.js:24]: It looks like it's only used [here](https://github.com/scality/backbeat/blob/375383c1b4651629349b550784cdcb3bdbd8369d/lib/Config.js#L200)

Which is used [here](https://github.com/scality/backbeat/blob/375383c1b4651629349b550784cdcb3bdbd8369d/lib/management/patchConfiguration.js#L38)

Which is used [here](https://github.com/scality/backbeat/blob/375383c1b4651629349b550784cdcb3bdbd8369d/lib/management/patchConfiguration.js#L70)

Appearently its used to patch configuration file which comes from [this ticket ](https://scality.atlassian.net/browse/BB-66)

Location type match comes from this PR : https://github.com/scality/backbeat/commit/a7035c1e5d04123f8fa125cf327dcaeff95a6758

And appearently there is a ticket BACKBEAT-3 which i don't have access to...

Comment (KillianG) [reply lib/Config.js:24]: It's only usefull for Zenko-1 because we are now using a config file for that so I guess I can remove this change

Comment (francoisferrand) [reply lib/Config.js:24]: that is not the end of the story I guess.

* We still have the code [to handle dynamic config changes, like with Orbit in Zenko 1], so the code must be updated still. It is a "feature" of the code base, even if not used currently by the product, and it **must** thus be maintained. (or we decide we won't need it, and remove it from the code base; but that is not the scope of this ticket).
* That said the question is not "where is the `locationTypeMatch` used", but where is the value (dmf, aws_s3...) defined/used. I.e. where/how is `config.bootstrapList[*].type` used

**Review by francoisferrand — APPROVED** [review]

**Pull request merged**